.circleci/config.yml

```diff
@@ -118,14 +117,19 @@ commands:
       - run:
           name: Push application with deployment vars
           command: |
-            cf push --vars-file << parameters.deploy_config_file >> \
+            cf push --strategy rolling \
+              --vars-file << parameters.deploy_config_file >> \
               --var AUTH_CLIENT_ID=${<< parameters.auth_client_id >>} \
               --var AUTH_CLIENT_SECRET=${<< parameters.auth_client_secret >>} \
               --var NEW_RELIC_LICENSE_KEY=${<< parameters.new_relic_license >>} \
               --var SESSION_SECRET=${<< parameters.session_secret >>} \
               --var HSES_DATA_FILE_URL=${<< parameters.hses_data_file_url >>} \
               --var HSES_DATA_USERNAME=${<< parameters.hses_data_username >>} \
               --var HSES_DATA_PASSWORD=${<< parameters.hses_data_password >>}
+      - run:
+          name: Push maintenance application
+          command: |
+            cd maintenance_page && cf push --vars-file ../<<parameters.deploy_config_file >>
```

Comment:
Really only need the `env` var from the vars-file. Wondering if we should pass that by itself? Felt like this was the most DRY way to do it, even though it passes a bunch of unnecessary vars to this push

Reply:
This looks like it's consistent with other `push `commands, even if it's only one var.